vendor/github.com/google/gopacket/layers/gtp.go

```diff
@@ -10,6 +10,7 @@ package layers
 import (
 	"encoding/binary"
 	"fmt"
+
```

Comment:
extra space

Reply:
It is the codes pull from gopacket by updating vendors.  It would be better to keep the spaces. Otherwise everytime we run "Make verify-vendor" in CICD, we will find updated files and the CICD will throw error.